x-pack/plugins/security_solution/cypress/e2e/cases/connectors.cy.ts

```diff
@@ -94,13 +100,10 @@ describe('Cases connectors', () => {
 
     cy.wait('@createConnector').then(({ response }) => {
       cy.wrap(response?.statusCode).should('eql', 200);
+      connectorId = response?.body.id ?? '';
```

Comment:
NIT: This `connectorId` does not seem to be used, can we remove it?

Reply:
Correct. I forgot about it. Done [6655997](https://github.com/elastic/kibana/pull/157254/commits/66559971ef5d23a17cf44a1616fd37b9d76302b7)